openstack_plugin/resources/compute/server.py

```diff
@@ -261,6 +255,29 @@ def _set_server_ips_runtime_properties(server):
     ctx.instance.runtime_properties['ipv4_addresses'] = ipv4_list
     ctx.instance.runtime_properties['ipv6_addresses'] = ipv6_list
 
+    # Set "ipv4_address" for server as runtime property
+    if server.access_ipv4:
+        # Maintain "access_ipv4" for backward compatibility
+        ctx.instance.runtime_properties['access_ipv4'] = server.access_ipv4
+        ctx.instance.runtime_properties['ipv4_address'] = server.access_ipv4
+
+    elif len(ipv4_list) == 1:
+        ctx.instance.runtime_properties['ipv4_address'] = ipv4_list[0]
+    else:
+        ctx.instance.runtime_properties['ipv4_address'] = None
```

Comment:
Let's not use `None` because it may cause problems to someone reading from intrinsic function.